azure/table_azure_cognitive_account.go

```diff
@@ -0,0 +1,407 @@
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAzureCognitiveAccount(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_cognitive_account",
+		Description: "Azure Cognitive Account",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AllColumns([]string{"name", "resource_group"}),
+			Hydrate:           getCognitiveAccount,
+			ShouldIgnoreError: isNotFoundError([]string{"ResourceNotFound", "ResourceGroupNotFound", "404"}),
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listCognitiveAccounts,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "Fully qualified resource ID for the resource.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromGo(),
+			},
+			{
+				Name:        "kind",
+				Description: "The kind of the resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "provisioning_state",
+				Description: "Gets the status of the cognitive services account at the time the operation was called. Possible values include: 'Accepted', 'Creating', 'Deleting', 'Moving', 'Failed', 'Succeeded', 'ResolvingDNS'.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.ProvisioningState"),
+			},
+			{
+				Name:        "type",
+				Description: "The type of the resource. E.g. 'Microsoft.Compute/virtualMachines' or 'Microsoft.Storage/storageAccounts'.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "custom_sub_domain_name",
+				Description: "The subdomain name used for token-based authentication.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.CustomSubDomainName"),
+			},
+			{
+				Name:        "date_created",
+				Description: "Gets the date of cognitive services account creation.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.DateCreated"),
+			},
+			{
+				Name:        "disable_local_auth",
+				Description: "Checks if local auth is disabled for the resource.",
+				Type:        proto.ColumnType_BOOL,
+				Default:     false,
+				Transform:   transform.FromField("Properties.DisableLocalAuth"),
+			},
+			{
+				Name:        "endpoint",
+				Description: "The endpoint of the created account.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.Endpoint"),
+			},
+			{
+				Name:        "etag",
+				Description: "The resource etag.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "is_migrated",
+				Description: "Checks if the resource is migrated from an existing key.",
+				Type:        proto.ColumnType_BOOL,
+				Default:     false,
```

Comment:
IsMigrated is bool type in API, does not need to mention the default value here.